Makefile.toml

```diff
@@ -196,6 +200,20 @@ fi
 # Args that will be passed into all testsys invocations.
 CARGO_MAKE_TESTSYS_ARGS = "${CARGO_MAKE_TESTSYS_KUBECONFIG_ARG}"
 
+TESTSYS_TEST_CONFIG_PATH = { script = [
+'''
+if [ -f "${TESTSYS_TEST_CONFIG_PATH}" ]; then
+   # If the config path exists
+   echo "${TESTSYS_TEST_CONFIG_PATH}"
+elif [ -f "${TESTSYS_FOLDER}/Test.toml" ]; then
+   # If the test config is in the TESTSYS_FOLDER it should be used.
+   echo "${TESTSYS_FOLDER}/Test.toml"
+else
+   echo "${TESTSYS_TEST_CONFIG_PATH}"
+fi
```

Comment:
Is this the right order of precedence? `Test.toml` from the root rather than the `tests` sub-directory? It's not consistent with `testsys.kubeconfig`.

Either way, I'd consider adding an assert that we don't have two `Test.toml` or `testsys.kubeconfig` files, one in the root and one in `tests`, since that's pretty likely to be an accident that will lead to unexpected behavior.

as a nit: `[ -s "${FILE}" ]` is often preferable to `-f`, because `-s` will also assert that the file is non-zero sized, which tends to correlate with usefulness.

Reply:
The reason for the order is to support a user having a single `TESTSYS_TESTS_DIR` but multiple `TEST.toml`s